.evergreen/legacy_config_generator/evergreen_config_lib/variants.py

```diff
@@ -76,6 +76,7 @@ def days(n: int) -> int:
             # Disable the MongoDB legacy shell download, which is not available in 5.0 for u22
             "SKIP_LEGACY_SHELL": "1"
         },
+        tags=["merge-gate"],
```

Comment:
```suggestion
        tags=["pr-merge-gate"],
```
To make it clearer what "merge" is referring to.